scheds/rust/scx_p2dq/src/bpf/main.bpf.c

```diff
@@ -56,24 +58,37 @@ u64 big_core_ids[MAX_CPUS];
 u64 dsq_time_slices[MAX_DSQS_PER_LLC];
 
 u64 max_exec_ns;
+u64 min_slice_ns = 500;
```

Comment:
Not super happy about needing this extra variable... will probably make everything ns and do the conversion in userspace rather than in `init`.